packages/tailwind/modules/breakpoint.ts

```diff
@@ -0,0 +1,40 @@
+type VuesticThresholdsList = Record<string, number>
+
+/**
+ * @param thresholds tailwind css screens settings
+ * @returns vuestic ui thresholds settings
+ */
+export const convertThresholds = (thresholds: Record<string, string>): VuesticThresholdsList => {
+  return Object.fromEntries(
+    Object.entries(thresholds)
+      .map(([key, value]) => {
+        const vuesticValue = Number(value.substring(0, value.length - 2))
+        return [key, vuesticValue]
+      }),
+  )
```

Comment:
Also is there any chance to get `NaN`?
```suggestion
  return Object.entries(thresholds).reduce((acc, [key, value]) => {
    acc[key] =  Number(value.substring(0, value.length - 2)) // what if NaN?
    return acc
  }, {} as VuesticThresholdsList)
```

Reply:
> Also is there any chance to get NaN?

 Only in case of typo. But will add a check just in case.